### DISCUSSION TITLE
regarding EC2 server to github remote repository

### DISCUSSION THREAD

**Discussion by stephensam02** [community/community#65975] [Programming Help]
I created in Ec2 instance in aws from ec2 server to git remote repository can't connect showing Authentication failed

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @stephensam02 ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by guilhermegobbo** [comment]
I usually use the ssh protocol to connect my github project to aws.

1 --> create an instance aws
2 --> connect the instance to superputty or another thing you want to use
3 --> create a ssh key to connect on github
4 --> clone the repository
5 --> run the codes you want
